rules/0580-win-security_rules.xml

```diff
@@ -1023,5 +1023,12 @@
     <description>IPsec Services encountered a potentially serious failure</description>
     <options>no_full_log</options>
   </rule>
+  
+   <rule id="60226" level="8">
```

Comment:
This event ID is contemplated at rule:
https://github.com/wazuh/wazuh-ruleset/blob/ed8396ecfb55d25b1cfb5f005171ddb99e05971e/rules/0580-win-security_rules.xml#L157

```xml
<rule id="60117" level="9">
    <if_sid>60103</if_sid>
    <field name="win.system.eventID">^517$|^1102$</field>
    <description>Windows audit log was cleared</description>
    <options>no_full_log</options>
    <group>logs_cleared,pci_dss_10.6.1,gpg13_10.1,gdpr_IV_35.7.d,</group>
</rule>
```

That rule is wrong due to the correct  `securityValue` is _INFORMATION_ instead of _AUDIT_SUCCESS_ so the parent rule should be the 60100, not 60103.

On the other side, to look for the event ID 507 is useless for this rule (it is equivalent to 1102 in Windows up to Vista, where the EventChannel is not supported).